File: broadcastreceiverlibrary/src/main/java/com/osapps/broadcastreceiverlibrary/sms/exceptions/SmsPdusException.java
package com.osapps.broadcastreceiverlibrary.sms.exceptions;

/**
 * Created by osapps on 09/05/2018.
 */
public class SmsPdusException extends SmsException {

    @Override
    protected String getErrorMsg() {
        return "sms pdus is null";
    }
}

File: broadcastreceiverlibrary/src/main/java/com/osapps/broadcastreceiverlibrary/sms/exceptions/SmsPdusException.kt
package com.osapps.broadcastreceiverlibrary.sms.exceptions

/**
 * Created by osapps on 09/05/2018.
 */
class SmsPdusException : SmsException() {

    override val errorMsg: String
        get() = "sms pdus is null"
}
